source/releasenotes/2024-05-21-5xx-error-messages.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Unbranded 5xx error messages launched
+published_date: "2024-05-21"
+categories: [general]
+---
+
+We’ve replaced all 5xx error messages with plain HTML and removed Pantheon branding for sites hosted on Pantheon. 
```

Comment:
[](url)
```suggestion
We’ve replaced with Pantheon branding on [all platform-generated 5xx error messages](guides/errors-and-server-responses/5xx-errors) with plain HTML. 
```

Reply:
typo, needs a leading `/` for the link to work

Reply:
fixed